src/factory.js

```diff
@@ -68,6 +69,8 @@ export class Factory {
     }
     this.renderQ = [];
     this.stave = null; // current stave
+
+    this.StaveNote = this.StaveNote.bind(this);
```

Comment:
By doing this we get the really elegant:  `[...noteStructs].map(vf.StaveNote)` that can be seen in the tests